release_scripts/Step2_Offline_PackageSigning.py

```diff
@@ -226,6 +226,105 @@ def getDstPath(pkgName, suffixStr='FileSuffix'):
 
 ################################################################################
 ################################################################################
+# Now verify multisig signatures before proceeding
+sys.path.append('/usr/lib/armory')
+from armoryengine.ALL import ADDRBYTE
+from jasvet import readSigBlock, verifySignature
+import urllib2
+import json
+import yaml # for assert file
```

Comment:
You may want to note that yaml and deb822 are external packages that need to be downloaded. I'm not sure if the best way to do it would be a comment here or a short README. I'm inclined to go with the latter but I have no strong preference at the moment.